comeet/UI/RoomsList/RoomsListViewController.swift

```diff
@@ -185,6 +207,10 @@ private extension RoomsListViewController {
         
         sliderView.setMinValue(CGFloat(startValue), maxValue: CGFloat(endValue))
         sliderView.setLeftValue(CGFloat(startAutoSelect), rightValue: CGFloat(endAutoSelect))
+        
+        let color = UIColor(red: 204.0/255.0, green: 204.0/255.0, blue: 1.0, alpha: 1.0)
+        let rangeImage = UIImage.from(color: color)
```

Comment:
We have a struct for UI constants mainly used for colors. Let's define this one there:
https://github.com/bookaroom/comeet-iOS/blob/master/comeet/UI/UIConstants.swift#L15-L17